src/Workspaces/Core/Portable/Simplification/SimplificationOptions.cs

```diff
@@ -45,11 +46,32 @@ public static class SimplificationOptions
         public static Option<bool> AllowSimplificationToBaseType { get; } = new Option<bool>(NonPerLanguageFeatureName, "AllowSimplificationToBaseType", true);
 
         /// <summary>
-        /// This option says if we should simplify away the this. or Me. in member access expression
+        /// This option says if we should simplify away the <see langword="this"/>. or <see langword="Me"/>. in member access expressions.
         /// </summary>
+        [Obsolete]
         public static PerLanguageOption<bool> QualifyMemberAccessWithThisOrMe { get; } = new PerLanguageOption<bool>(PerLanguageFeatureName, "QualifyMemberAccessWithThisOrMe", defaultValue: false);
```

Comment:
Personally, as far as the UI is concerned, I'd rather pretend the old setting doesn't exist. If someone sets the setting programmatically then we can make that the setting for the 4 specific settings to maintain API compatibility, but for the UI a top-level checkbox with 4 checkboxes underneath feels overkill.